widget/playground/src/containers/FunctionalLayout/FunctionalLayout.From.tsx

```diff
@@ -0,0 +1,40 @@
+import type { ChangeEvent } from 'react';
+
+import { Divider, TextField, Typography } from '@rango-dev/ui';
+import React from 'react';
+
+import { useConfigStore } from '../../store/config';
+import { DefaultChainAndToken } from '../DefaultChainAndToken';
+
+import { FromAmount, FromToContainer } from './FunctionalLayout.styles';
+
+export function FromSection() {
+  const {
+    config: { amount },
+    onChangeAmount,
+  } = useConfigStore();
+
+  const handleChangeAmount = (e: ChangeEvent<HTMLInputElement>) => {
+    onChangeAmount(parseFloat(e.target.value || '0'));
+  };
+
+  return (
+    <>
+      <FromToContainer>
+        <DefaultChainAndToken type="Source" />
```

Comment:
Is it possible to show a tool-tip for default token to guide user select a blockchain first while it is disabled?

Reply:
fixed.